### DISCUSSION TITLE
(Extremly lowest bottom priority) Rename "coal" in piratespeak

### DISCUSSION THREAD

**Discussion by Jmooroof** [TeamGalacticraft/Galacticraft#230] [Feature Requests]
Coal in piratespeak is fuel, which can get confused with fuel.

**Comment by Adi-0** [comment]
100% agree o7

**Comment by Roelymole** [comment]
Arguably worse, the word iron gets replaced with "steel" in the pirate speak translation, which also causes confusion between compressed iron and compressed steel, for example.

As Marcus pointed out in #464, we should probably think of something different for the Galacticraft translations.

> I think it would be better for Galacticraft pirate speak to rename (real) steel/fuel to something else, as other mods may name their iron "steel", etc. However, I must admit that I have no clue what to rename them to.

Any good ideas?
